doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -737,11 +737,15 @@ Libraries for networking
 
 * :ref:`lib_nrf_cloud_fota` library:
 
-* Updated:
+  * Updated:
+
+    * The :kconfig:option:`CONFIG_NRF_CLOUD_FOTA_DOWNLOAD_FRAGMENT_SIZE` Kconfig option is made available and used also when the :kconfig:option:`CONFIG_NRF_CLOUD_FOTA_POLL` Kconfig option is enabled.
```

Comment:
```suggestion
    * The :kconfig:option:`CONFIG_NRF_CLOUD_FOTA_DOWNLOAD_FRAGMENT_SIZE` Kconfig option to be available and used also when the :kconfig:option:`CONFIG_NRF_CLOUD_FOTA_POLL` Kconfig option is enabled.
```
I realize these are existing entries, but the entries are meant to be read with the containing bullet, so this is "Updated the Kconfig option to", and so on.